index.html

```diff
@@ -27,6 +27,148 @@ <h1>Juan Pablo Guerrero Rios</h1>
+            <div class="projects1b3">
+                <div>
+                    <h3 class="tittle3">Finish mobile section 2</h3>
+                
+                <div class="b3">
+                    <button class="sbutton">HTML/CSS</button>
+                    <button class="sbutton">Ruby on Rails</button>
+                    <button class="sbutton">JavaScript</button>
+                </div>
+
+                <div>
+                <button class="start1">See this project →</button>
+                </div>
+            </div> 
+        </section>
+
+        <section id="Projects">
+            
+            <img class="lap1" src="./images/lap3.svg">
+            <div class="projects1b3">
+                <div>
+                    <h3 class="tittle3">Finish mobile section 3</h3>
+                
+                <div class="b3">
+                    <button class="sbutton">HTML/CSS</button>
+                    <button class="sbutton">Ruby on Rails</button>
+                    <button class="sbutton">JavaScript</button>
+                </div>
+
+                <div>
+                <button class="start1">See this project →</button>
+                </div>
+            </div> 
+        </section>
+
+        <section id="Projects">
+            
+            <img class="lap1" src="./images/lap4.svg">
+            <div class="projects1b3">
+                <div>
+                    <h3 class="tittle3">Finish mobile section 4</h3>
+                
+                <div class="b3">
+                    <button class="sbutton">HTML/CSS</button>
+                    <button class="sbutton">Ruby on Rails</button>
+                    <button class="sbutton">JavaScript</button>
+                </div>
+
+                <div>
+                <button class="start1">See this project →</button>
+                </div>
+            </div> 
+        </section>
+
+        <section id="Projects">
+            
+            <img class="lap1" src="./images/lap5.svg">
+            <div class="projects1b3">
+                <div>
+                    <h3 class="tittle3">Finish mobile section 5</h3>
+                
+                <div class="b3">
+                    <button class="sbutton">HTML/CSS</button>
+                    <button class="sbutton">Ruby on Rails</button>
+                    <button class="sbutton">JavaScript</button>
+                </div>
+
+                <div>
+                <button class="start1">See this project →</button>
+                </div>
+            </div> 
+        </section>
+
+        <section id="greetings">
+            <h1>About</h1>
+            <p class="firstp">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Atque doloribus maiores,
+                 rem excepturi et exercitationem commodi dolore, veritatis reprehenderit ullam quas, nemo obcaecati
+                  perferendis quisquam sunt placeat praesentium? Nobis, nihil.</p>
+            <button class="start2">Get my CV</button> 
```

Comment:
- Kindly make sure you `get my cv ` button arranges in the same line as the all container to make it  you can add the width of the button to 325px :100: 
## Your design

![image](https://user-images.githubusercontent.com/95297251/198752043-027e9273-2c38-4b2d-9247-22a308fedffc.png)

## Figma design 

![image](https://user-images.githubusercontent.com/95297251/198752493-3b8fbfc7-8f8d-4b21-8a1f-35c95f26f7f8.png)

Reply:
Hi Magana, hope youa re very well. In this case I tried what yoy siggested me, but it did not work, so I set the margin-left of the button to 16px, and it is now starting at the same margin of the parragraph and the title. Can I subimit this way?